web3/providers/eth_tester/defaults.py

```diff
@@ -23,46 +36,61 @@
     excepts,
 )
 
+from web3.types import (
+    LogReceipt,
+    RPCResponse,
+    TxReceipt,
+)
+
+if TYPE_CHECKING:
+    from eth_tester import (  # noqa: F401
+        EthereumTester,
+    )
+
 
-def not_implemented(*args, **kwargs):
+def not_implemented(*args: Any, **kwargs: Any) -> NoReturn:
     raise NotImplementedError("RPC method not implemented")
 
 
 @curry
-def call_eth_tester(fn_name, eth_tester, fn_args, fn_kwargs=None):
+def call_eth_tester(
+    fn_name: str, eth_tester: "EthereumTester", fn_args: Any, fn_kwargs: Any=None
+) -> RPCResponse:
     if fn_kwargs is None:
         fn_kwargs = {}
     return getattr(eth_tester, fn_name)(*fn_args, **fn_kwargs)
 
 
-def without_eth_tester(fn):
+def without_eth_tester(fn: Callable[..., Any]) -> Callable[..., RPCResponse]:
     # workaround for: https://github.com/pytoolz/cytoolz/issues/103
     # @functools.wraps(fn)
-    def inner(eth_tester, params):
+    def inner(eth_tester: "EthereumTester", params: Any) -> Callable[..., RPCResponse]:
         return fn(params)
     return inner
 
 
-def without_params(fn):
+def without_params(fn: Callable[..., Any]) -> Callable[..., RPCResponse]:
     # workaround for: https://github.com/pytoolz/cytoolz/issues/103
     # @functools.wraps(fn)
-    def inner(eth_tester, params):
+    def inner(eth_tester: "EthereumTester", params: Any) -> Callable[..., RPCResponse]:
         return fn(eth_tester)
     return inner
 
 
 @curry
-def preprocess_params(eth_tester, params, preprocessor_fn):
+def preprocess_params(
+    eth_tester: "EthereumTester", params: Any, preprocessor_fn: Callable[..., Any]
+) -> Tuple["EthereumTester", Callable[..., Any]]:
     return eth_tester, preprocessor_fn(params)
 
 
-def static_return(value):
-    def inner(*args, **kwargs):
+def static_return(value: Any) -> Callable[..., Any]:
```

Comment:
This one should be doable as `def static_return(value: TValue) -> Callable[..., TValue]:`